CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+### Intentional Omission of Identity Validation
+
+Note that this CIP does not mention any kind of identity validation or verification for the members of the Constitutional Committee or the DReps.
+
+This is intentional.
+
+We hope that the community will strongly consider only voting for and delegating to those who provide something like a DID to make themselves known.
+However, enforcing identity verification is very difficult without some centralized oracle, which we consider to be a step in the wrong direction.
+
+### Initial Implementation Stage
+
+This document describes an ambitious change to Cardano governance. A two-stage implementation process will
+allow it to be rolled out more quickly, with some of the more complicated aspects
+(DReps and constitutional changes) rolled out in a second stage. This will also allow time
+for fuller evaluation of/consultation on incentives and other significant  issues, while establishing the basis for
+governance in Voltaire.
+Below is a high-level summary of an implementation plan for (very roughly) half of the plan.
+
+1. Add a subset of (important) governance actions to the transaction body (while deprecating protocol parameter updates and MIR certificates).
+  The following governance actions will be supported initially:
+  * protocol parameters
+  * hard forks
+  * withdrawals from the treasury
+  * Constitutional Committee changes
+2. Add votes to the transaction body, but disallow DRep votes.
+3. The first version of the new `RATIFY` rule will only count Constitutional Committee votes, plus SPO votes for hard forks (similarly to the previously defined, and now obsoleted, `CIP-47`).
+
+The remainder of this proposal will be implemented in the second stage.
+
+### Piggybacking on stake pool stake distribution
+
+The Cardano protocol is based on a Proof-of-Stake consensus mechanism, so using a stake-based governance approach is sensible. However, there are many ways by which one could define how to record the stake distribution between participants. As a reminder, network addresses can currently contain two sets of credentials: one to identify who can unlock funds at an address (a.k.a. payment credentials) and one that can be delegated to a stake pool (a.k.a. delegation credentials).
+
+Rather than defining a third set of credentials, we instead propose to re-use the existing delegation credentials, using a new on-chain certificate to determine the governance stake distribution. This implies that the DReps can (and likely will) differ from SPOs, so creating balance. On the flip side, it means that the governance stake distribution suffers from the same shortcomings as that for block production: for example, wallet software providers must support multi-delegation schemes and must facilitate the partitioning of stake into sub-accounts should an Ada holder desire to delegate to multiple DReps.
```

Comment:
That's fair. It's for checks and balances, so that the tricameral government isn't really just a single branch. SPOs have their own concerns and viewpoints, which may not always be in complete agreement with everyone else, and it is important to have a way for other viewpoints to be represented.

Reply:
> I think, as I previous mentioned, that this can be generalized. It is not only SPOs, but many different kind of stakeholders that has their own PoVs and agends; developers, investors, SPOs, etc.
> 
> I think this needs to be baked into the system somehow. I don't think one person can or should represent on all issues (withstanding a few, extremely invested people or entities, who will probably be in the CC)

Well you can always experiment and not do a Tricameral form of government as Mr. Corduan has suggested and expand into a pentacameral or heptacameral. At that point you may need weighted voting systems or % towards systems. For example, with a pentacameral you have 5 organizations with 20% of the vote and within that, each organization gets their own approval percentage. Lets say all 5 approve 12%, 8%, 20%, 11%, and 1%. When totaling the pentacameral vote you would have an approval percent of 52% so in a majority system then this initiative is passed. You can further delegate voting power to sub-groups created from the main pentacameral system. All comes down to how decentralized you want to get, if it can be coded, how you will combat fraud in a more complex system, and how efficient you want your government to be.

Reply:
One argument for DReps to be different from the SPOs would be that SPOs probably like higher fees than most other groups. This could be balanced out by people running businesses, who probably prefer lower fees (and have significant voting power). In general, there are all sorts of biases that might pop up by linking staking & voting in some direct manner.

However, I'd expect that most SPOs would also register as DReps, and that most people would give their voting power to their stake pool. So I'd still expect that a large chunk of both distributions to have the same shape.

Reply:
> However, I'd expect that most SPOs would also register as DReps, and that most people would give their voting power to their stake pool. So I'd still expect that a large chunk of both distributions to have the same shape.

I agree with this sentiment. The union between the two sets would be (very) large IMO, or at least I'd argue that logic dictates it would, given reasons I stated in another comment.

Reply:
> can you explain why we would bake this into the protocol itself, and not just rely on the diversity of DReps?

Well, if one system has to rely on the diversity of one group to represent the rest of the community but you have 2 other systems that you have to meet X requirements to partake in, then it is easier for a predetermined group of people to meet those requirements and then that predetermined group of people who come from X background have more weight/voting power naturally.

That predetermined group of people also probably have the same interests and will vote the same way. The community that has to nominate DReps has sadly more of a diversity of thought and thus will have candidates that will either become totalitarian if a majority arises within that voting block or have candidates that simply are lame ducks and will fold to the power of the other 2 groups. In other words, the diversity of the DReps is their weakness because there are no requirements to partake in governance unlike the other 2 voting blocks. The SPO's have a knowledge requirement and the constitutional committee has a wealth requirement. Instead of having these initial requirements to vote for different groups, you need to either A. Have a group that meets all these requirements or B. have a group that meets none of these requirements or C. Have a baseline requirement for all groups. C. is probably the best answer to me. Once you have established C. there will be infighting, This infighting will result in sub groups. In order to prevent a bicameral system within SPO's, The Constitutional Committee, and DReps, you need people to be able to form their own committees and groups THAT REQUIRE their own requirements to join the group. Thus the process described above can fragment and fragment to the point of a truly decentralized system where people vote bottom up because they actually meet the requirements to vote bottom up.

It is pivotal that this fragmentation occurs in every branch of the tricameral system. A perfectly balanced political system should be able to flip upside down and still function justly.

It is easy to establish in a top down system a mechanism of voting that allows everyone, it is hard to establish in a bottom up system a mechanism of voting that allows everyone to vote. Thus, you need to change the requirements in a bottom up system as you go higher and higher in the system.

Reply:
Sorry, I do not respond quickly. I work during the week and still feel as if I should be paid for this.

Reply:
I think this: https://github.com/cardano-foundation/CIPs/pull/380#discussion_r1032780310

And what I have typed are the same ideas just not entirely flushed out @GGAlanSmithee @JaredCorduan

Reply:
In addition to this and my comment that "A perfectly balanced political system should be able to flip upside down and still function justly"

This means that what is even being voted on should be able to change. The people at the bottom have their own opinions and reasons for voting, but those opinions should be able to replace the parameters at the top (AKA the original proposers, proposal). Instead of a proposal being rigid and people having to vote on a set statement such as "Let's hardfork for x reason", the individual voter should be able to substitute their reason for voting with the statement so now it reads "Let's hardfor for y reason" instead. This creates TWO votes. One vote to keep the original proposal or substitute it with the new one and a second vote to even pass the thing. To form the most perfect voting structure, this must be an option @JaredCorduan

Reply:
I saw an interesting video by army of spies about chat.openai.com. I was inspired by the platforms outcome when asked to write a 4 paragraph academic essay comparing and contrasting the theories of nationalism of Benedit Anderson and Ernest Geller. In 10 seconds a very decent essay popped out. 

I think the huge legacy of human endeavour to today contains all the wisdom to construct a very inspiring Constitution we need.

If this AI was asked to list the top 12 points for Human prosperity from the Zoroastrian Avesta, The Hindu Ramayana and Mahabharata, The Buddhist Tipitaka,, The Bible, The Koran The Tora etc, plus as many Legal rights and Constitutions of the world, US , Magna Carta etc we would see something very powerful maybe.

Reinventing the wheel is not required. AI's impartial, non conformation biased, non sectarian, multicultural, historic and very fast analysis  might give us something very balanced and a valuable framework to draw upon. Cardano could enable us to breathe life into the old and for the 1st time ever limit malevolent manipulation and interpretation of the great gifts to humanity from the greatest ever hearts and minds of the last 5000 years to the benifit of all.

Reply:
As @WhiteHorseAvatar says it is very fast analysis but the issue I have with AI is when it comes to the definitions of words. An AI ultimately creates a definition through analyzing a ton of words and finding the average definition. So when an AI is asked to list the top 12 points for human prosperity, it has to assign definitions (inputs) to those words and then match outputs to how closely it fits those definitions.

Now that I have that gripe out of the way, please expand on your post or how AI can be useful in actually writing constitutional documents or whatnot. @WhiteHorseAvatar Should we generate an AI based constitution or legal code to maybe take inspiration from?

Reply:
>  Have a baseline requirement for all groups. C. is probably the best answer to me.

Could you give a concrete suggestion for how to accomplish this? Remember, we are not considering any form of KYC or identity management, and I also consider legal restrictions completely out of scope in this proposal.

---

> "Let's hardfork for x reason", the individual voter should be able to substitute their reason for voting with the statement so now it reads "Let's hardfor for y reason" instead

This is not something which is at all enforceable on-chain, I would not want to complicate the protocol to accommodate this.

---

I will add a section about what topics I consider to be off-topic. In particular, the contents of the constitution is off topic (it's extremely important, but out of scope here).

Reply:
Well, if identity management includes like what Atala prism is doing then no, I can not think of an example. If it includes what they are doing then yeah, you can do anything at that point. A voting system based off of an education requirement, membership requirement, time on chain requirement, transactions made requirement, credit requirement.

To my second point, "Let's hardfork for x reason", the individual voter should be able to substitute their reason for voting with the statement so now it reads "Let's hardfor for y reason" instead. This is not something which is at all enforceable on-chain, I would not want to complicate the protocol to accommodate this.

I would like this on chain but you do not have to do this on chain. This can be coded using like rust or something and then the final result can be put on chain as the actual issue to vote on after x amount of time or after it passes x threshold of popular votes or whatever. @JaredCorduan

Reply:
> A voting system based off of an education requirement, membership requirement, time on chain requirement, transactions made requirement, credit requirement.

Including this kind of information (such as in the DRep registration metadata) is great. But enforcing it is difficult without introducing an oracle or other centralizing mechanism.

> This can be coded using like rust or something and then the final result can be put on chain as the actual issue to vote on after x amount of time or after it passes x threshold of popular votes or whatever.

This sounds like it could be accomplished by re-submitting the governance action with new metadata.